labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -32,8 +32,12 @@ object CollectionExercise01 {
    * Case 3: so it is okay if you want to just give up
    *
    */
+  // It's a lie. O doesn't actually map to e, it maps to k. Great.
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val code = "ejp mysljylc kd kxveddknmc re jsicpdrysirbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcdde kr kd eoya kw aej tysr re ujdr lkgc jva z"
+    val answer = "our language is impossible to understandthere are twenty six factorial possibilitiesso it is okay if you want to just give upy q"
+    val translator: Map[Char, Char] = (code zip answer).foldLeft(Map.empty[Char, Char])((m, p) => m + p)
```

Comment:
I definitely have a lot of these inconsistencies in my code, but is there any particular reason you used `.foldLeft` but not `.zip`?